src/evidence/stepping_stone.jl

```diff
@@ -25,7 +25,9 @@ function stepping_stone_pair(log_sum_ratios::GroupBy)
     return (estimator1, -estimator2) 
 end
 
-function stepping_stone(input)
-    pair = stepping_stone_pair(input) 
+function stepping_stone(pt::PT, ::NonReversiblePT)
+    pair = stepping_stone_pair(pt) 
     return (pair[1] + pair[2]) / 2.0
-end
\ No newline at end of file
+end
+
+stepping_stone(pt::PT, ::VariationalPT) = error() # todo
```

Comment:
How about using just use one of the legs, e.g. the variational one? Are you worried about cases where the variational reference has a different support compared to the target? We could just add a warning when we encounter that situation.

Reply:
This was just because I wasn't sure what the stepping_stone() was calculating with the recorder when we have two legs. I'm leaving it as an error for now (so that it doesn't display in the reports) but can address this in a follow-up issue. I will need to check with you, though, that my implementation is correct.